packages/react-components/src/components/Toast.spec.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { render, fireEvent } from '../test-utils';
+import { Toast, Variants } from './Toast';
+
+const customClass = 'my-css-class';
+const toastContent = 'This example content';
+const mockedFunction = jest.fn();
+const mockedFunctionOnClose = jest.fn();
+const toastAction = {
+  label: 'Example action',
+  handler: mockedFunction,
+};
+
+describe('<Banner> component', () => {
+  it('should allow for custom class', () => {
+    const { container } = render(
+      <Toast className={customClass}>{toastContent}</Toast>
+    );
+
+    expect(container.firstChild).toHaveClass(customClass);
+  });
+
+  it('should render as info by default', () => {
+    const { container } = render(<Toast>{toastContent}</Toast>);
+
+    expect(container.firstChild).toHaveClass('lc-toast--info');
+  });
+
+  it('should render as success', () => {
+    const { container } = render(
+      <Toast variant={Variants.Success}>{toastContent}</Toast>
+    );
+
+    expect(container.firstChild).toHaveClass('lc-toast--success');
+  });
+
+  it('should render as warning', () => {
+    const { container } = render(
+      <Toast variant={Variants.Warning}>{toastContent}</Toast>
+    );
+
+    expect(container.firstChild).toHaveClass('lc-toast--warning');
+  });
+
+  it('should render as error', () => {
+    const { container } = render(
+      <Toast variant={Variants.Error}>{toastContent}</Toast>
+    );
+
+    expect(container.firstChild).toHaveClass('lc-toast--error');
+  });
+
+  it('should render as notification', () => {
+    const { container } = render(
+      <Toast variant={Variants.Notification}>{toastContent}</Toast>
+    );
+
+    expect(container.firstChild).toHaveClass('lc-toast--notification');
+  });
+
+  it('should render with action button', () => {
+    const { getByTestId } = render(
+      <Toast action={toastAction}>{toastContent}</Toast>
+    );
+
+    expect(getByTestId('actionButton')).toBeTruthy();
```

Comment:
It would be better to use `.toBeVisible` from jest-dom. It's more explicit and checks more things.